mapstory/templates/layers/layer_data.html

```diff
@@ -114,17 +114,13 @@
                 <h3>Map Services</h3>
                 <div class="embed">WMS 
                     <i class="fa fa-question-circle ng-scope" tooltip-append-to-body="true" tooltip-placement="top" tooltip="OpenGIS® Web Map Service Interface Standard"></i>
-                  
-                    <input id="shareWMS" value ="http://{{ request.get_host }}/geoserver/geonode/{{ resource }}/wms?REQUEST=GetCapabilities&SERVICE=WMS&TILED=true&VERSION=1.1.1">
-                  
+                    <input id="shareWMS" value ="https://{{ request.get_host }}/geoserver/geonode/{{ resource }}/wms?REQUEST=GetCapabilities&SERVICE=WMS&TILED=true&VERSION=1.1.1"> 
                     <button class="copyclip btn" data-clipboard-target="#shareWMS"><i class="fa fa-clipboard" aria-hidden="true"></i></button>
                 </div>
-
                 <div class="embed"> WFS
                     <i class="fa fa-question-circle ng-scope" tooltip-append-to-body="true" tooltip-placement="top" tooltip="OGC® Web Feature Service"></i>
-                    <input id="shareWFS" value ="http://{{ request.get_host }}/geoserver/geonode/{{ resource }}/wfs?REQUEST=GetCapabilities&SERVICE=WFS&VERSION=2.0.0">
+                    <input id="shareWFS" value ="https://{{ request.get_host }}/geoserver/geonode/{{ resource }}/wfs?REQUEST=GetCapabilities&SERVICE=WFS&VERSION=2.0.0">
```

Comment:
Should use request.protocol rather than explicitly using https so that it will be consistent in non-ssl environments.